src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java

```diff
@@ -121,6 +138,22 @@ protected void writeColumnSpec(StringBuilder builder, SinkRecordField f) {
   }
 
   protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+          builder.append(value);
```

Comment:
I was hoping to leave this to the relevant JDBC driver or database. The db's decimal type may have a limited precision/scale, and JDBC allows for `PreparedStatement.setBigDecimal()` where the `BigDecimal` has its own precison/scale, I trust there needs to be some mapping/validation there... perhaps something to exercise in a system test.